openrdp/scripts/bootscan.py

```diff
@@ -215,9 +260,17 @@ def execute(self, triplet):
                         rec_name = trp_names.pop(i)
                         parents = trp_names
                         if val != 0.0:
-                            self.raw_results.append((rec_name, parents, *event, val))
+                            raw_results.append((rec_name, parents, *event, val))
+
+        return raw_results
+
+    def execute_all(self, total_combinations, seq_names):
+        self.seq_names = seq_names
+        self.total_triplet_combinations = total_combinations
+        with multiprocessing.Pool(self.np) as p:
+            results = p.map(self.execute, enumerate(generate_triplets(self.align)))
```

Comment:
Last part of merge conflict.  @dpannguyen moved a conditional statement out of the loop over `trp_names`.  @GopiGugan is this compatible with your changes?

Reply:
Yes, that doesn't conflict with the changes I made